Core/clim/input-editing-drei.lisp

```diff
@@ -164,31 +166,66 @@ activated with GESTURE"))
 ;;; rely on internal features and implement input-editor support in
 ;;; CLIM-INTERNALS.
 
+;;; When yanking history of commands on the stream, Following helper function
+;;; ensures the sublists are not evaluated by quoting them. They are
+;;; not meant to be evaluated anyway. (See comment on "define-command" in
+;;; Core/clim-core/commands.lisp).
+
+(defun %quote-sublists (list)
+  (let ((result nil))
+    (loop for item in list
+	  do (if (listp item)
+		 (if (eql (car item) 'quote)
+		     (push item result)
+		     (push `(quote ,item) result))
+		 (push item result)))
+    (reverse result)))
+
+(defun %object-command-p (object type)
+  (when (and (equal type 'command-or-form)
+		     (listp object)
+		     (alexandria:starts-with-subseq "COM-" (symbol-name `,(car object))))
+    t))
```

Comment:
please just make a single when inside history-yank-*, having a separate function for that is an overkill

Reply:
As you wish! 
Just realized that part needs to be more intelligent, because commands are not guaranteed to start with "COM-", it can be customized by the user.